assignment/api/src/resources/assignment.properties

```diff
@@ -275,7 +275,8 @@ gen.hideduedate = Hide due date from students
 grad2 = Grade entered ''{0}'' is larger than the maximum grade allowed ''{1}''. Please make a correction or click the original button again to save the entered value ''{0}''.
 
 grad3 = Graded
-
+grade.title=Grades
+grade.delimiter=.
```

Comment:
Never put delimiters into translation files.

Check the formattedText.getDecimalSeparator() method.